src/EditorFeatures/VisualBasicTest/Diagnostics/AbstractVisualBasicDiagnosticProviderBasedUserDiagnosticTest.vb

```diff
@@ -47,5 +49,64 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics
         Protected Overrides Function GetLanguage() As String
             Return LanguageNames.VisualBasic
         End Function
+
+#Disable Warning BC40003 ' Member shadows an overloadable member declared in the base type
```

Comment:
💡 This warning should be addressed, I believe by using `Shadows`. Also, I think there is a code fix for this case.